cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/templates/etl/batch/BaseETLBatchTest.java

```diff
@@ -52,7 +54,7 @@ public static void setupTest() throws IOException {
       DBSource.class, KVTableSource.class, StreamBatchSource.class, TableSource.class);
     addTemplatePlugins(TEMPLATE_ID, "batch-sinks-1.0.0.jar",
       BatchCubeSink.class, DBSink.class, KVTableSink.class,
-      TableSink.class, TimePartitionedFileSetDatasetAvroSink.class);
+      TableSink.class, TimePartitionedFileSetDatasetAvroSink.class, AvroKeyOutputFormat.class, AvroKey.class);
```

Comment:
Yes, they were added in a separate PR https://github.com/caskdata/cdap/pull/2480